collections/blog.item

```diff
@@ -15,7 +15,7 @@
   </h1>
 
   {.main-image?}
-  <img class="blog-article__main-image" src="{assetUrl}?w=1000w">
+  <img class="blog-article__main-image" alt="" role="presentation" src="{assetUrl}?w=1000w">
```

Comment:
I take it there's no facility to add alt text for the main image in Squarespace? Seems a shame.